packages/TesterApp/src/ui/types.ts

```diff
@@ -0,0 +1 @@
+export type WithChildren<T> = React.PropsWithChildren<T>;
```

Comment:
Since this type is just an alias for built-in React type wouldn't it be better to use it explicitly?